dotnet/src/Extensions/Planning.StepwisePlanner/StepwisePlanner.cs

```diff
@@ -37,7 +37,8 @@ public class StepwisePlanner
     public StepwisePlanner(
         IKernel kernel,
         StepwisePlannerConfig? config = null,
-        string? prompt = null)
+        string? prompt = null.
+        string? config = null)
```

Comment:
1. It won't compile. There is dot instead of comma in line 40, and also there are two `config` parameters in this method.
2. We should think whether to accept `string` or `PromptTemplateConfig` for prompt configuration. If we use strong type instead of string, it will reduce the number of errors in string parsing logic.

Reply:
Yes, I agree. Passing PromptTemplateConfig, could be good idea, instead of string to reduce the error. Updated the PR